pandas/core/internals/managers.py

```diff
@@ -15,7 +15,7 @@
 
 import numpy as np
 
-from pandas._config import get_option
+from pandas._config.config import _global_config
```

Comment:
is this much faster than using pandas._config.config.options?

Reply:
It's in any case faster, since `_global_config` is just a plain python dict (which is quite fast), while the options object is a custom class mimicking attribute access:

```
In [32]: %timeit _global_config["mode"]["copy_on_write"]
52.6 ns ± 1.85 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)

In [36]: %timeit pd._config.config.options.mode.copy_on_write
2.78 µs ± 22 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)
```

Strangely it seems that this is even slower than the `pd.get_option("mode.copy_on_write")`:

```
In [37]: %timeit pd.get_option("mode.copy_on_write")
1.38 µs ± 14.6 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
```

Reply:
If we really want to avoid a private import, this is also an option:

```
In [39]: %timeit pd.options.d["mode"]["copy_on_write"]
85.9 ns ± 2.39 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each)
```

Reply:
not worth tying ourself in knots over, thanks for checking